zzz_modular_syzygy/code/modules/loadout/lists/uniforms.dm

```diff
@@ -24,3 +24,7 @@
 	display_name = "black turtleneck"
 	path = /obj/item/clothing/under/syndicate/civilian
 
+/datum/gear/uniform/tacticalturtleneck
```

Comment:
Non-blocking change; I would suggest moving this to its own file in `zzzz_modular_occulus` rather than editing `zzz_modular_syzygy`

Reply:
the syzygy folder is technically one of our modular folders anyway. So this is fine. It'll end up getting moved eventually when I get to it as I'm slowly moving those files to our folder anyway.